TestHelpers/TestHelpers.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.1</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>

```

Comment:
You should probably keep this as netstandard2.1.  No reason to update to 6 and it will only mean it can be used in less places.